tests/test_data_source.py

```diff
@@ -0,0 +1,329 @@
+from datetime import datetime
+import pytest
+from th2_data_services.exceptions import EventNotFound
+from th2_data_services_lwdp.commands import grpc as commands
+from th2_data_services_lwdp.data_source import GRPCDataSource
+from th2_data_services_lwdp.streams import Streams
+
+DEMO_HOST = "de-th2-qa"
+DEMO_PORT = "31038"
+
+def test_find_events_by_id_from_data_provider():
+	data_source = GRPCDataSource(f"{DEMO_HOST}:{DEMO_PORT}")
+
+	expected_event = {
+		'eventId': '1ca84723-f946-11ec-80f0-83883d31de94',
+		'parentEventId': '104eaf10-f904-11ec-80f0-83883d31de94',
+		'startTimestamp': {
+			'epochSecond': 1656684006,
+			'nano': 546878000
+		},
+		'endTimestamp': {
+			'epochSecond': 1656684006,
+			'nano': 546880000
+		},
+		'eventName': 'Service [arfq01dc03] emitted event with status INFO',
+		'eventType': 'Service event',
+		'body': [{
+			'data': 'Initiated logon request',
+			'type': 'message'
+		}],
+		'isBatched': False,
+		'status': 'SUCCESS',
+		'attachedMessageId': []
+	}
+
+	expected_events = []
+	expected_events.append(expected_event)
+	expected_events.append(
+		{
+			'eventId': '1946f3f4-f946-11ec-9065-197dc41e2de9',
+			'parentEventId': 'ffeb1fcc-f903-11ec-9065-197dc41e2de9',
+			'startTimestamp': {
+				'epochSecond': 1656684000,
+				'nano': 875141000
+			},
+			'endTimestamp': {
+				'epochSecond': 1656684000,
+				'nano': 875143000
+			},
+			'eventName': 'Service [arfq02dc10] emitted event with status INFO',
+			'eventType': 'Service event',
+			'body': [{
+				'data': 'Initiated logon request',
+				'type': 'message'
+			}],
+			'isBatched': False,
+			'status': 'SUCCESS',
+			'attachedMessageId': []
+		}
+	)
```

Comment:
It looks weird. Why do not use the following? 
expected_events = [
  expected_event,
  {}
]